app/helpers/static-text.ts

```diff
@@ -156,6 +173,20 @@ export const settingsText = {
     tooltip:
       'Prevents this mod from showing warnings in the Mod Manager when starting the game',
   },
+  showAlphaSettings: {
+    label: 'Show Alpha Settings',
+    tooltip: '',
+  },
+  alphaPath: {
+    label: 'Alpha game folder (contains alpha game exe)',
```

Comment:
this text is a bit too big to fit comfortably in a min width window, maybe move the "contains alpha game exe" thing to the description